docs/reference/index.md

```diff
@@ -1,5 +1,11 @@
 # Welcome to HACL Packages' documentation!
 
+HACL Packages provides ready-to-use crypto algorithms based on the formally verified [HACL\* crypto distribution].
```

Comment:
Not sure what best to use for "HACL*" but it's not a distribution.
Another option would be something like "based on the formally verified code from the HACL* repository". That's the most neutral form.

```suggestion
HACL Packages provides ready-to-use crypto algorithms based on the formally verified [HACL\*].
```